packages/paste-core/components/table/stories/index.stories.tsx

```diff
@@ -0,0 +1,364 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Box} from '@twilio-paste/box';
+import {MediaObject, MediaBody, MediaFigure} from '@twilio-paste/media-object';
+import {Menu, MenuButton, MenuItem, MenuSeparator, useMenuState} from '@twilio-paste/menu';
+import {AttachIcon} from '@twilio-paste/icons/esm/AttachIcon';
+import {MoreIcon} from '@twilio-paste/icons/esm/MoreIcon';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Table, THead, TBody, TFoot, Tr, Td, Th} from '../src';
+
+const MoreMenu: React.FC<{}> = () => {
+  const menu = useMenuState();
+  return (
+    <Box display="flex" justifyContent="flex-end">
+      <MenuButton {...menu} tableLayout="reset" size="reset">
```

Comment:
Not sure this is supposed to be `tableLayout`. It's given you some funky blue icon buttons

![image](https://user-images.githubusercontent.com/368249/95267283-b9bedd80-07e9-11eb-8415-96eb6b82aedb.png)

Reply:
Damn find/replace all!